examples/run_mock_client_main.cc

```diff
@@ -2,52 +2,136 @@
   // TODO(tugan): add support to listen on host:port based on configuration
-  std::string server_address("0.0.0.0:50051");
+  std::string master_address("0.0.0.0:50051");
+  std::string chunk_server_address("0.0.0.0:50052");
   auto credentials = grpc::InsecureChannelCredentials();
-  auto master_channel = grpc::CreateChannel(server_address, credentials);
+  auto master_channel = grpc::CreateChannel(master_address, credentials);
+  auto chunk_server_lease_channel =
+      grpc::CreateChannel(chunk_server_address, credentials);
   MasterMetadataServiceClient metadata_client(master_channel);
+  ChunkServerServiceClient chunk_server_client(chunk_server_lease_channel);
 
-  // Prepare a mock gRPC request and client context
+  // Prepare a mock gRPC: OpenFile
   OpenFileRequest open_request;
-  grpc::ClientContext client_context1;
   open_request.set_filename("/tmp/test_open_file");
   open_request.set_chunk_index(0);
   open_request.set_mode(OpenFileRequest::WRITE);
   open_request.set_create_if_not_exists(true);
 
-  // Send the request and print the response to screen
-  StatusOr<OpenFileReply> status_or =
-      metadata_client.SendRequest(open_request, client_context1);
-  std::cout << "Request sent: \n" << open_request.DebugString() << std::endl;
-
-  if (status_or.ok()) {
-    std::cout << "Response received: \n"
-              << status_or.ValueOrDie().DebugString() << std::endl;
-  } else {
-    std::cout << "Request failed: \n"
-              << status_or.status().ToString() << std::endl;
-  }
+  grpc::ClientContext client_context;
+  LogRequestAndResponse(
+      open_request, metadata_client.SendRequest(open_request, client_context));
 
+  // Prepare a mock gRPC: DeleteFile
   DeleteFileRequest delete_request;
   delete_request.set_filename("/tmp/test_delete_file");
+
   grpc::ClientContext client_context2;
   Status status = metadata_client.SendRequest(delete_request, client_context2);
-  std::cout << "Request sent: \n" << delete_request.DebugString() << std::endl;
-  std::cout << "Request status: \n" << status.ToString() << std::endl;
+  LOG(INFO) << "Request sent: " << delete_request.DebugString();
+  LOG(INFO) << "Request status: " << status.ToString();
+
+  // Prepare a mock gRPC: GrantLease
+  google::protobuf::Timestamp lease_expiration_time;
+  lease_expiration_time.set_seconds(1000);
+  GrantLeaseRequest grant_lease_request;
+  grant_lease_request.set_chunk_handle("/tmp/test_grant_lease");
+  grant_lease_request.set_chunk_version(1);
+  *grant_lease_request.mutable_lease_expiration_time() = lease_expiration_time;
+
+  grpc::ClientContext client_context3;
+  LogRequestAndResponse(
+      grant_lease_request,
+      chunk_server_client.SendRequest(grant_lease_request, client_context3));
+
+  // Prepare a mock gRPC: RevokeLease
+  RevokeLeaseRequest revoke_lease_request;
+  revoke_lease_request.set_chunk_handle("/tmp/test_revoke_lease");
+  *revoke_lease_request.mutable_original_lease_expiration_time() =
```

Comment:
1. yes, there's not setter for a message inside another message
2. made it slightly easier instead of commenting it:

```
grant_lease_request.mutable_lease_expiration_time()->set_seconds(1000);
```